### ISSUE TITLE
Crash when provided size is too small

### ISSUE THREAD

**Issue by Zenthae** [linebender/druid#1582]
On windows, when the provided size is too small, it panic when starting

```rust
        .window_size((200., 20.))
        .window_size_policy(WindowSizePolicy::User);
```

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value'
```

**Comment by Ciantic** [comment]
I'm looking in to this, I need this myself too as I want to create transparent windows the size of tooltip.

**Comment by PoignardAzur** [comment]
@Ciantic Progress?

**Comment by cmyr** [comment]
Reported fixed in #1575, can reopen if someone hits it again.
